src/bootstrap4-floating-label.scss

```diff
@@ -76,6 +84,7 @@ $label-top-position-lg: #{- $label-height-lg + $label-height-lg / 2.5}; // -30px
   // see https://github.com/mui-org/material-ui/blob/v3.3.2/packages/material-ui/src/NativeSelect/NativeSelect.js#L77
   // See ["the mouse event go "through" the element and target whatever is "underneath" that element instead"](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events)
   pointer-events: none;
+  @extend %label-rtl;
```

Comment:
`@extend %label-rtl` inside `label-inside-input-common` is not enough:

![Screen Shot 2019-10-10 at 21 01 34](https://user-images.githubusercontent.com/643434/66598367-313f3a00-eba1-11e9-93fc-5d3e93c42093.png)

We have to add:
```SCSS

// Support for "right to left" languages
[dir='rtl'] {
  %label-inside-input-rtl {
    left: auto !important;
    right: 0;
  }
  %label-above-input-rtl {
    left: auto !important;
    right: $input-padding-x-sm;
  }
}
```
and
```SCSS

@mixin label-above-input-common {
  ...

  @extend %label-above-input-rtl;
}
```

in order to fix `<select>`:

![Screen Shot 2019-10-10 at 21 02 56](https://user-images.githubusercontent.com/643434/66598447-5b90f780-eba1-11e9-9e45-8b1cf36d7213.png)

`label-inside-input-common` becomes:
```SCSS
@mixin label-inside-input-common {
  ...

  @extend %label-inside-input-rtl;
}
```

Notice also the small space on the right of the label "input RTL":

![Screen Shot 2019-10-10 at 21 08 28](https://user-images.githubusercontent.com/643434/66598801-233de900-eba2-11e9-9b77-264aa1829520.png)

This is achieved with `right: $input-padding-x-sm`